dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -995,6 +995,25 @@ private void verifyImageChecksumsAreSaved(WorkflowVersion version) {
         );
     }
 
+    @Test
+    public void testGettingImagesFromDockerHub() {
+        final ApiClient webClient = getWebClient(USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowsApi = new WorkflowsApi(webClient);
+        final io.dockstore.openapi.client.ApiClient openAPIClient = getOpenAPIWebClient(USER_2_USERNAME, testingPostgres);
+        Ga4Ghv20Api ga4Ghv20Api = new Ga4Ghv20Api(openAPIClient);
+
+        // Test that a version of an official dockerhub image will get an image per architecture. (python 2.7) Also check that regular
+        // DockerHub images are grabbed correctly broadinstitute/gatk:4.0.1.1
+        WorkflowVersion version = snapshotWorkflowVersion(workflowsApi, "dockstore-testing/broad-prod-wgs-germline-snps-indels", DescriptorType.WDL.toString(), "/JointGenotypingWf.wdl", "1.1.2");
+        WorkflowVersion version2 = snapshotWorkflowVersion(workflowsApi, "NatalieEO/broad-prod-wgs-germline-snps-indels", DescriptorType.WDL.toString(), "/JointGenotypingWf.wdl", "1.1.2");
```

Comment:
Consider putting this into dockstore-testing

Reply:
deleted, shouldn't be there